.github/workflows/docker.yml

```
@@ -17,6 +17,9 @@ on:
   workflow_dispatch:
   release:
     types: [published, edited]
+    permissions:
```

actionlint (1.7.4)

unexpected key "permissions" for "release" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)